test/gtest/fused_conv_bias_res_add_activ.cpp

```diff
@@ -25,14 +25,44 @@
  *******************************************************************************/
 #include <gtest/gtest.h>
 #include <miopen/miopen.h>
+#include <miopen/env.hpp>
+#if MIOPEN_USE_COMPOSABLEKERNEL
+#include <miopen/solver/ck_utility_common.hpp>
+#endif
 
 #include "tensor_util.hpp"
 #include "get_handle.hpp"
 
 #include "conv3d_test_case.hpp"
 
+MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_TEST_ALL)
+MIOPEN_DECLARE_ENV_VAR_STR(MIOPEN_TEST_FLOAT_ARG)
+
+#if MIOPEN_USE_COMPOSABLEKERNEL
```

Comment:
This switch does not seem useful given how we rely heavily on CK and that we don't have (to my knowledge) alternatives to CK that intend to replace it. Do we expect to ship/deploy MIOpen in a setting where CK is absent or does not work? 

CC: @JehandadKhan , @junliume

Reply:
This is for CI stage 'Fp32 Hip Debug NOCK AnyGPU Build-Only' to pass.